content/services/services_powerdev.rst

```diff
@@ -4,6 +4,9 @@ PowerLinux / OpenPOWER Development Hosting
 :title: Development Hosting
 :menu: Services, Development Hosting, 3
 
+.. |br| raw:: html
+
+    <br />
```

Comment:
This should have been taken out, I was doing something wrong earlier and this was a quick fix that wasn't even necessary.